concourse/pipelines/guest-package-build.yaml

```diff
@@ -1727,3 +2130,14 @@ groups:
   - promote-artifact-registry-apt-transport-staging
   - promote-artifact-registry-el-plugins-stable
   - promote-artifact-registry-apt-transport-stable
+- name: compute-image-windows
+  jobs:
+  - build-compute-image-windows
+  - upload-windows-certgen-staging
+  - upload-windows-auto-updater-staging
+  - upload-windows-powershell-staging
+  - upload-windows-sysprep-staging
+  - promote-windows-certgen-stable
+  - promote-windows-auto-updater-stable
+  - promote-windows-powershell-stable
+  - promote-windows-sysprep-stable
```

Comment:
nit but is there a reason the IDEs our windows engineers use tend not to end files with newline?

Reply:
This is the default behavior for VSCode when editing yaml.